src/Slider/Slider.js

```diff
@@ -98,9 +99,9 @@ class Slider extends PureComponent {
     }
   }
 
-  reMapValues(reversed) {
-    const { values } = this.state
-    return this.setValues(values.map(d => d.val), reversed)
+  reMapValues(reversed, values) {
+    // if values was not passed, fall back to using state
+    return this.setValues(values || this.state.values.map(d => d.val), reversed)
```

Comment:
Test coverage doesn't cover the control flow where values from props is undefined and reMapValues falls back to using state. My initial reaction is that maybe we don't need to fall back onto state at all. But I'm not sure.

Reply:
A hint for how to cover this in tests would be welcome :), passing undef values props seems to not be the right way to accomplish this

Reply:
This looks good.  I see what you mean about the tests.  I'll fix that up.  I was going to fix some things on the tests anyways.